frontend/src/components/base/NavDesktopUserMenu.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <v-menu offset-y dark
+          right content-class="ec-usermenu"
+          transition="slide-y-transition"
+          z-index="5">
+    <template v-slot:activator="{ on,value }">
+      <v-toolbar-items>
+        <v-btn right text
+               :class="{ 'v-btn--open': value }" v-on="on">
+          <span class="sr-only-sm-and-down">
+            {{ username }}
+          </span>
+          <v-icon class="ma-2">mdi-account</v-icon>
+        </v-btn>
+      </v-toolbar-items>
+    </template>
+    <v-list dense class="user-nav"
+            light color="blue-grey lighten-5">
+      <v-list-item block :to="{ name: 'profile' }">
+        <v-icon left>mdi-account</v-icon>
+        <span>Profil</span>
+      </v-list-item>
+      <v-list-item block
+                   exact :to="{ name: 'camps', params: { groupName: encodeURI('Pfadi Bewegung Schweiz') } }">
+        <v-icon left>mdi-format-list-bulleted-triangle</v-icon>
+        <span>Meine Camps</span>
+      </v-list-item>
+      <v-list-item block @click="logout">
+        <v-icon v-if="logoutIcon" left>{{ logoutIcon }}</v-icon>
+        <v-progress-circular v-else indeterminate
+                             size="18" class="mr-2" />
+        <span>Log out</span>
+      </v-list-item>
+    </v-list>
+  </v-menu>
+</template>
+
+<script>
+
+export default {
+  name: 'NavDesktopUserMenu',
+  data () {
+    return {
+      logoutIcon: 'mdi-logout'
+    }
+  },
+  methods: {
```

Comment:
```suggestion
  props: {
    username: { type: String, required: true }
  },
  methods: {
```

Reply:
Added as a computed. Doesn't need to be a prop, because it isn't used in NavDesktop.